tests/shell/check_kdb_internal_check.sh

```diff
@@ -22,61 +22,30 @@ if contains "$KDB" "full" || contains "$KDB" "static"; then
 	ACTUAL_PLUGINS=$ADDED_PLUGINS_WITHOUT_ONLY_SHARED
 fi
 
-printf "Checking %s\n" "$ACTUAL_PLUGINS"
+printf "Checking %s\n\n" "$ACTUAL_PLUGINS"
 
 for PLUGIN in $ACTUAL_PLUGINS; do
 	ARGS=""
 	case "$PLUGIN" in
-	'jni')
-		# References:
-		#  - https://travis-ci.org/sanssecours/elektra/builds/410641048
-		#  - https://issues.libelektra.org/1466
-		#  - https://issues.libelektra.org/1963
-		continue
-		;;
-	"tracer")
-		# output on open/close
-		continue
-		;;
-	"timeofday")
-		# output on open/close
-		continue
-		;;
-	"counter")
-		# output on open/close
-		continue
-		;;
-	"xfconf")
```

Comment:
From the builds it looks like we will have to disable at least `xfconf`.

Reply:
Ah okay, so it doesn't work during packaging. I'll see if I can disable it only at that time somehow...